src/days-2-4-adv/adv.py

```diff
@@ -33,12 +37,18 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
+rock = Item("Rock", "This is a rock")
+
+room['outside'].addItem(rock)
+
 #
 # Main
 #
 
 # Make a new player object that is currently in the 'outside' room.
 
+
+
```

Comment:
Would it be a good idea to write a Main class to start the game?
for example. (pseudocode)
class Main
    method start game()
   
    <logic>

end class

Main instance
main instance start game()